style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: 'Inter';
+  src:
+    url('./fonts/Inter/Inter-Regular.woff') format('woff'),
+    url('./fonts/Inter/Inter-Regular.woff2') format('woff2');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Inter';
+  src:
+    url('./fonts/Inter/Inter-Bold.woff') format('woff'),
+    url('./fonts/Inter/Inter-Bold.woff2') format('woff2');
+  font-weight: bold;
+  font-style: normal;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Inter, 'Times New Roman', Times, serif;
+  color: #344563;
+  background-color: #fff;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.header > * {
+  margin: 0 40px;
+  flex-shrink: 0;
+  text-decoration: none;
+  color: inherit;
+}
+
+.header .logo {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+main .content {
+  background-image: url("./images/background.svg");
+  background-repeat: no-repeat;
+  background-position: top center;
+  background-size: cover;
```

Comment:
[OPTIONAL]

- [ ] I suggest you use `background-size: contain;` 💡  , this will help the background image styling much better 😄